### ISSUE TITLE
add copy feature

### ISSUE THREAD

**Issue by llydd** [LanceMoe/openai-translator#10]
If there could be an added copy button, that would be even better.

**Comment by HarperLoop** [comment]
Agreed, as well as a quick clear input field button (for large texts after translation)

**Comment by LanceMoe** [comment]
Ok, I'll implement this feature asap~
